### DISCUSSION TITLE
Add a way to remove the "+" button in the bottom middle portion of the screen.

### DISCUSSION THREAD

**Discussion by N6624** [qnblackcat/uYouPlus#1167] [Ideas]
I just updated to v18.14.1-3.0 from v18.08.1_2.3.1 and a + button near the bottom of the screen appeared. Since I do not create content I do not need or want this button to be there. Is there a way to remove it right now or would it be possible to add it as a feature?

**Comment by yodaluca23** [comment]
uYou Settings> Hide Tabs> Hide Create Tab

**Reply by N6624** [reply]
Thank you. I feel so stupid now for not looking under the uYou tab.
